shared/css/barlowSemiCondensed-font.scss

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: 'Barlow Semi Condensed Semibold';
+  font-style: normal;
+  font-weight: 600;
+  src: url('/fonts/barlowSemiCondensed/BarlowSemiCondensed-SemiBold.ttf') format('truetype'),
+local('?');
+}
+
+@font-face {
+  font-family: 'Barlow Semi Condensed Medium';
+  font-style: normal;
+  font-weight: 500;
+  src: url('/fonts/barlowSemiCondensed/BarlowSemiCondensed-Medium.ttf') format('truetype'), local('?');
+}
+
+$barlowSemiCondensed-semibold: 'Barlow Semi Condensed Semibold', sans-serif;
+$barlowSemiCondensed-medium: 'Barlow Semi Condensed Medium', sans-serif;
```

Comment:
I think these variables can be added here since we already import `font.scss` in a lot of places: https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/font.scss — @madelynkasula what are your thoughts?

Reply:
yeah good catch, kelby -- i think moving these variables to `font.scss` makes more sense